src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceManagementService.java

```diff
@@ -39,4 +46,16 @@ public UploadResponse upload(List<MultipartFile> files) {
         }
     }
 
+    public byte[] download(String documentId) {
```

Comment:
ensure junit coverage here

Reply:
There are already several tests in uk.gov.hmcts.reform.sscs.service.EvidenceManagementServiceTest which cover this.